src/components/siteContent/SiteContent.js

```diff
@@ -59,7 +48,12 @@ const SiteContent = ({ group1, group2, group3 }) => {
                 onMouseLeave={onMouseLeave}
               />
               {isDescriptionOpened && index === Number(isPhotoHovered) && (
-                <div className={styles.description}>{isPhotoHovered}</div>
+                <div className={styles.description}>
+                  <p style={{ fontWeight: "bold" }}>
+                    {groups[isPhotoHovered].description}
```

Comment:
isPhotoFovered to number? Jeżeli tak to trzeba zmienić nazwę bo sugeruje boolean 😄